src/controller/python/chip/clusters/CHIPClusters.py

```diff
@@ -946,7 +946,9 @@ def ReadAttribute(self, device: ctypes.c_void_p, cluster: str, attribute: str, e
         if not func:
             raise UnknownAttribute(cluster, attribute)
         funcCaller = self._ChipStack.Call if imEnabled else self._ChipStack.CallAsync
-        funcCaller(lambda: func(device, endpoint, groupid))
+        res = funcCaller(lambda: func(device, endpoint, groupid))
+        if res != 0:
```

Comment:
can just be `if res:`

Simpler logic is easier to read.

Reply:
Done